src/main/webapp/app/exam/manage/exams/exam-checklist-component/exam-checklist-exercisegroup-table/exam-checklist-exercisegroup-table.component.html

```diff
@@ -0,0 +1,39 @@
+<table class="table table-striped">
+    <thead>
+        <tr>
+            <th style="max-width: 100px">
+                <span>Exercise Group</span>
+            </th>
+            <th>
+                <span>Exercise</span>
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr *ngFor="let exerciseGroup of exerciseGroups">
+            <td style="max-width: 100px">
+                {{ exerciseGroup.title }}
+            </td>
+            <td>
+                <table class="table table-striped">
+                    <thead>
+                        <tr>
+                            <th>
+                                <span>Exercise</span>
+                            </th>
+                            <th>
+                                <span>Number of participations </span>
+                            </th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr *ngFor="let exercise of exerciseGroup.exercises">
+                            <td class="d-none d-md-table-cell">{{ exercise.title }}</td>
+                            <td class="d-none d-md-table-cell">{{ exercise.numberOfParticipations }}</td>
+                        </tr>
```

Comment:
Here you could maybe add the points of the exercise somewhere

Reply:
Good idea, thanks. I did it in the latest commit!